tests/scenarios/v2-addon-dev-typescript-test.ts

```diff
@@ -100,6 +104,9 @@ appScenarios
           ], {
             mapFilename: (name) => reexportMappings[name] || name,
           }),
+          addon.appReexports(['initializers/**/*.js'], {
```

Comment:
ya know, I never thought of just using `appReexports` twice. 🤔

is there a risk that the other copy of this plugin would overwrite / erase the work of the other plugin?

Reply:
I don't think so, given that we merge `app-js` with original. It also does `emitFile`, which is fine to call many times. In case you have duplicate files processing with different `appReexports` calls the rollup will give you a warning